README

```diff
@@ -1878,7 +1873,9 @@ state     - RTE state machine
 Miscellaneous frameworks:
 -------------------------
 
+allocator   - Memory allocator
 backtrace   - Debugging call stack backtrace support
+btl         - Point-to-point Byte Transfer Layer
 compress    - Compression algorithms
 crs         - Checkpoint and restart service
```

Comment:
Meh -- forgot to `git clean -dfx` between v1.10.x and v2.x checkout.  Fixed now.